frontend/.eslintrc.json

```diff
@@ -34,15 +38,13 @@
       "as-needed"
     ],
     "no-alert": 0,
-    "linebreak-style": 0,
-    "max-len": [
+    "@stylistic/max-len": [
       "error",
       {
         "code": 120
       }
     ],
     "prefer-object-spread": 0,
-    "function-call-argument-newline": 0,
```

Comment:
Same for this? Maybe good to give a one-line reason for everything removed or newly added